src/discord-cluster-manager/cogs/leaderboard_cog.py

```diff
@@ -791,8 +799,8 @@ async def get_task_template(
                 )
                 await send_discord_message(
                     interaction,
-                    f"Task `{leaderboard_name}` does not have a template for `{lang}`.\n"
-                    f"Choose from:\n{langs}",
+                    f"Leaderboard `{leaderboard_name}` does not have a template for language `{lang}`.\n"  # noqa: E501
```

Comment:
I  wouldn't use the word language here; e.g., if we can have "inline-cuda" as a template, which isn't really  a language